### PR TITLE
Misc updates

### PR DESCRIPTION
**Pull request by charasyn**
Changes:
- Add level names and fix volume issues
- Adjust high score ranking and initial scores
- EOL object can go to any specified scene: This adds an EOL object to Level1 going to the Thanks For Playing screen
- Fade out volume between levels and adjust volume

### CODE REVIEW

File: godotproject/EOL.gd [hunk 1]
```diff
@@ -8,5 +11,5 @@ func _on_EOL_body_entered(body):
 	if body.name == "Player" and not triggered:
 		triggered = true
 		var level_name = Global.get_current_scene_property('level_name')
-		var next_path = "res://Level" + str(int(get_tree().current_scene.name) + 1) + ".tscn"
+		# var next_path = "res://Level" + str(int(get_tree().current_scene.name) + 1) + ".tscn"
```
Comment (fheysel) [inline godotproject/EOL.gd]: delete this if its not needed

File: godotproject/Menus/ThanksForPlayingScreen/ThanksForPlayingScreen.gd [hunk 2]
```diff
@@ -0,0 +1,10 @@
+extends MarginContainer
+
```
Comment (fheysel) [inline godotproject/Menus/ThanksForPlayingScreen/ThanksForPlayingScreen.gd:2]: Has this been linked to the end of The Big City level?

Comment (charasyn) [reply godotproject/Menus/ThanksForPlayingScreen/ThanksForPlayingScreen.gd:2]: Yep

**Review by fheysel — APPROVED** [review]
Looks good, I love the new level names! A few comments and questions but feel free to merge.

File: godotproject/EOL.gd (hunk 1 above)
Comment (charasyn) [inline godotproject/EOL.gd]:
```suggestion
```

File: godotproject/Level1.tscn [hunk 3]
```diff
@@ -337,3 +338,7 @@ position = Vector2( 5992, -296 )
 [node name="Level1BacktrackPlayer" type="AudioStreamPlayer" parent="."]
 stream = ExtResource( 15 )
 autoplay = true
+
+[node name="EOL" parent="." instance=ExtResource( 17 )]
+position = Vector2( 10050, 0 )
+next_path = "res://Menus/ThanksForPlayingScreen/ThanksForPlayingScreen.tscn"
```
Comment (charasyn) [inline godotproject/Level1.tscn:344]: Linking of "The Big City" to the Thanks For Playing screen

File: godotproject/Menus/ThanksForPlayingScreen/ThanksForPlayingScreen.gd (hunk 2 above)
Comment (fheysel) [reply godotproject/Menus/ThanksForPlayingScreen/ThanksForPlayingScreen.gd:2]: nice!

**Pull request merged**